[libxslt] Build failure on arm64-osx

Package: libxslt[core,python]:arm64-osx -> 1.1.42

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 16.0.0.16000026
-    vcpkg-tool version: 2999-12-31-unknownhash
    vcpkg-scripts version: 4ff3a6d329 2025-01-24 (24 hours ago)

**To Reproduce**

`vcpkg install --enforce-port-checks --allow-unsupported`
**Failure logs**

```
-- Using cached Fix-package-version.patch.
-- Using cached GNOME-libxslt-v1.1.42.tar.gz.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/GNOME-libxslt-v1.1.42.tar.gz
-- Applying patch python3.patch
-- Applying patch msvc-no-suffix.patch
-- Applying patch libexslt-pkgconfig.patch
-- Applying patch fix-gcrypt-deps.patch
-- Applying patch skip-install-docs.patch
-- Applying patch /Users/leanderSchulten/git_projekte/vcpkg/downloads/Fix-package-version.patch
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean
-- Found external ninja('1.12.1').
-- Configuring arm64-osx
CMake Warning at vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:344 (message):
  The following variables are not used in CMakeLists.txt:

      PYTHON_EXECUTABLE

  Please recheck them and remove the unnecessary options from the
  `vcpkg_cmake_configure` call.

  If these options should still be passed for whatever reason, please use the
  `MAYBE_UNUSED_VARIABLES` argument.
Call Stack (most recent call first):
  ports/libxslt/portfile.cmake:37 (vcpkg_cmake_configure)
  scripts/ports.cmake:196 (include)


-- Building arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.30.1-osx/cmake-3.30.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j11
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/install-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/libxslt/portfile.cmake:51 (vcpkg_cmake_install)
  scripts/ports.cmake:196 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/install-arm64-osx-dbg-out.log</summary>

```
Change Dir: '/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg'

Run Build Command(s): /opt/homebrew/bin/ninja -v -v -j11 install
[1/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/namespaces.c.o -MF CMakeFiles/LibXslt.dir/libxslt/namespaces.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/namespaces.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/namespaces.c
[2/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/attrvt.c.o -MF CMakeFiles/LibXslt.dir/libxslt/attrvt.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/attrvt.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/attrvt.c
[3/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/functions.c.o -MF CMakeFiles/LibXslt.dir/libxslt/functions.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/functions.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/functions.c
[4/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/extensions.c.o -MF CMakeFiles/LibXslt.dir/libxslt/extensions.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/extensions.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/extensions.c
[5/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/attributes.c.o -MF CMakeFiles/LibXslt.dir/libxslt/attributes.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/attributes.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/attributes.c
[6/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/extra.c.o -MF CMakeFiles/LibXslt.dir/libxslt/extra.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/extra.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/extra.c
[7/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/imports.c.o -MF CMakeFiles/LibXslt.dir/libxslt/imports.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/imports.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/imports.c
[8/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/documents.c.o -MF CMakeFiles/LibXslt.dir/libxslt/documents.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/documents.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/documents.c
[9/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/keys.c.o -MF CMakeFiles/LibXslt.dir/libxslt/keys.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/keys.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/keys.c
[10/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/numbers.c.o -MF CMakeFiles/LibXslt.dir/libxslt/numbers.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/numbers.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/numbers.c
[11/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/pattern.c.o -MF CMakeFiles/LibXslt.dir/libxslt/pattern.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/pattern.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/pattern.c
[12/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/security.c.o -MF CMakeFiles/LibXslt.dir/libxslt/security.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/security.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/security.c
[13/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/xsltlocale.c.o -MF CMakeFiles/LibXslt.dir/libxslt/xsltlocale.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/xsltlocale.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/xsltlocale.c
[14/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -DLIBXSLT_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibExslt.dir/libexslt/common.c.o -MF CMakeFiles/LibExslt.dir/libexslt/common.c.o.d -o CMakeFiles/LibExslt.dir/libexslt/common.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libexslt/common.c
[15/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/templates.c.o -MF CMakeFiles/LibXslt.dir/libxslt/templates.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/templates.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/templates.c
[16/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/variables.c.o -MF CMakeFiles/LibXslt.dir/libxslt/variables.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/variables.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/variables.c
[17/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/preproc.c.o -MF CMakeFiles/LibXslt.dir/libxslt/preproc.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/preproc.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/preproc.c
[18/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/xsltutils.c.o -MF CMakeFiles/LibXslt.dir/libxslt/xsltutils.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/xsltutils.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/xsltutils.c
[19/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -DLIBXSLT_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibExslt.dir/libexslt/crypto.c.o -MF CMakeFiles/LibExslt.dir/libexslt/crypto.c.o.d -o CMakeFiles/LibExslt.dir/libexslt/crypto.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libexslt/crypto.c
[20/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/xslt.c.o -MF CMakeFiles/LibXslt.dir/libxslt/xslt.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/xslt.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/xslt.c
[21/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -DLIBXSLT_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibExslt.dir/libexslt/exslt.c.o -MF CMakeFiles/LibExslt.dir/libexslt/exslt.c.o.d -o CMakeFiles/LibExslt.dir/libexslt/exslt.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libexslt/exslt.c
[22/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -DLIBXSLT_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibExslt.dir/libexslt/dynamic.c.o -MF CMakeFiles/LibExslt.dir/libexslt/dynamic.c.o.d -o CMakeFiles/LibExslt.dir/libexslt/dynamic.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libexslt/dynamic.c
[23/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -DLIBXSLT_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibExslt.dir/libexslt/date.c.o -MF CMakeFiles/LibExslt.dir/libexslt/date.c.o.d -o CMakeFiles/LibExslt.dir/libexslt/date.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libexslt/date.c
[24/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -DLIBXSLT_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibExslt.dir/libexslt/functions.c.o -MF CMakeFiles/LibExslt.dir/libexslt/functions.c.o.d -o CMakeFiles/LibExslt.dir/libexslt/functions.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libexslt/functions.c
[25/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -DLIBXSLT_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibExslt.dir/libexslt/saxon.c.o -MF CMakeFiles/LibExslt.dir/libexslt/saxon.c.o.d -o CMakeFiles/LibExslt.dir/libexslt/saxon.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libexslt/saxon.c
[26/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibXslt.dir/libxslt/transform.c.o -MF CMakeFiles/LibXslt.dir/libxslt/transform.c.o.d -o CMakeFiles/LibXslt.dir/libxslt/transform.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libxslt/transform.c
[27/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -DLIBXSLT_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibExslt.dir/libexslt/sets.c.o -MF CMakeFiles/LibExslt.dir/libexslt/sets.c.o.d -o CMakeFiles/LibExslt.dir/libexslt/sets.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libexslt/sets.c
[28/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -DLIBXSLT_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibExslt.dir/libexslt/strings.c.o -MF CMakeFiles/LibExslt.dir/libexslt/strings.c.o.d -o CMakeFiles/LibExslt.dir/libexslt/strings.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libexslt/strings.c
[29/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBXML_STATIC -DLIBXSLT_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -fPIC -MD -MT CMakeFiles/LibExslt.dir/libexslt/math.c.o -MF CMakeFiles/LibExslt.dir/libexslt/math.c.o.d -o CMakeFiles/LibExslt.dir/libexslt/math.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/libexslt/math.c
[30/36] : && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.30.1-osx/cmake-3.30.1-macos-universal/CMake.app/Contents/bin/cmake -E rm -f libxslt.a && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ar qc libxslt.a  CMakeFiles/LibXslt.dir/libxslt/attributes.c.o CMakeFiles/LibXslt.dir/libxslt/attrvt.c.o CMakeFiles/LibXslt.dir/libxslt/documents.c.o CMakeFiles/LibXslt.dir/libxslt/extensions.c.o CMakeFiles/LibXslt.dir/libxslt/extra.c.o CMakeFiles/LibXslt.dir/libxslt/functions.c.o CMakeFiles/LibXslt.dir/libxslt/imports.c.o CMakeFiles/LibXslt.dir/libxslt/keys.c.o CMakeFiles/LibXslt.dir/libxslt/namespaces.c.o CMakeFiles/LibXslt.dir/libxslt/numbers.c.o CMakeFiles/LibXslt.dir/libxslt/pattern.c.o CMakeFiles/LibXslt.dir/libxslt/preproc.c.o CMakeFiles/LibXslt.dir/libxslt/security.c.o CMakeFiles/LibXslt.dir/libxslt/templates.c.o CMakeFiles/LibXslt.dir/libxslt/transform.c.o CMakeFiles/LibXslt.dir/libxslt/variables.c.o CMakeFiles/LibXslt.dir/libxslt/xslt.c.o CMakeFiles/LibXslt.dir/libxslt/xsltlocale.c.o CMakeFiles/LibXslt.dir/libxslt/xsltutils.c.o && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib libxslt.a && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.30.1-osx/cmake-3.30.1-macos-universal/CMake.app/Contents/bin/cmake -E touch libxslt.a && :
[31/36] : && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.30.1-osx/cmake-3.30.1-macos-universal/CMake.app/Contents/bin/cmake -E rm -f libexslt.a && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ar qc libexslt.a  CMakeFiles/LibExslt.dir/libexslt/common.c.o CMakeFiles/LibExslt.dir/libexslt/crypto.c.o CMakeFiles/LibExslt.dir/libexslt/date.c.o CMakeFiles/LibExslt.dir/libexslt/dynamic.c.o CMakeFiles/LibExslt.dir/libexslt/exslt.c.o CMakeFiles/LibExslt.dir/libexslt/functions.c.o CMakeFiles/LibExslt.dir/libexslt/math.c.o CMakeFiles/LibExslt.dir/libexslt/saxon.c.o CMakeFiles/LibExslt.dir/libexslt/sets.c.o CMakeFiles/LibExslt.dir/libexslt/strings.c.o && /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/ranlib libexslt.a && /Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.30.1-osx/cmake-3.30.1-macos-universal/CMake.app/Contents/bin/cmake -E touch libexslt.a && :
[32/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBEXSLT_STATIC -DLIBXML_STATIC -DLIBXSLT_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/python -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -isystem /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -MD -MT CMakeFiles/LibXsltMod.dir/python/types.c.o -MF CMakeFiles/LibXsltMod.dir/python/types.c.o.d -o CMakeFiles/LibXsltMod.dir/python/types.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/python/types.c
[33/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBEXSLT_STATIC -DLIBXML_STATIC -DLIBXSLT_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/python -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -isystem /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -MD -MT CMakeFiles/LibXsltMod.dir/python/libxslt.c.o -MF CMakeFiles/LibXsltMod.dir/python/libxslt.c.o.d -o CMakeFiles/LibXsltMod.dir/python/libxslt.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/python/libxslt.c
[34/36] ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBEXSLT_STATIC -DLIBXML_STATIC -DLIBXSLT_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/python -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -isystem /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -MD -MT CMakeFiles/LibXsltMod.dir/libxslt-py.c.o -MF CMakeFiles/LibXsltMod.dir/libxslt-py.c.o.d -o CMakeFiles/LibXsltMod.dir/libxslt-py.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg/libxslt-py.c
FAILED: CMakeFiles/LibXsltMod.dir/libxslt-py.c.o 
ccache /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc -DLIBEXSLT_STATIC -DLIBXML_STATIC -DLIBXSLT_STATIC -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean/python -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg -I/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/src/v1.1.42-1a87331e7c.clean -isystem /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include/libxml2 -isystem /opt/homebrew/opt/python@3.13/Frameworks/Python.framework/Versions/3.13/include/python3.13 -fPIC -Wall -Wextra -Wshadow -Wpointer-arith -Wcast-align -Wwrite-strings -Waggregate-return -Wstrict-prototypes -Wmissing-prototypes -Wnested-externs -Winline -Wredundant-decls -Wno-long-long -Wno-format-extra-args -g -arch arm64 -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX15.2.sdk -MD -MT CMakeFiles/LibXsltMod.dir/libxslt-py.c.o -MF CMakeFiles/LibXsltMod.dir/libxslt-py.c.o.d -o CMakeFiles/LibXsltMod.dir/libxslt-py.c.o -c /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg/libxslt-py.c
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg/libxslt-py.c:171:5: error: call to undeclared function 'xsltCalibrateAdjust'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  171 |     xsltCalibrateAdjust(delta);
      |     ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg/libxslt-py.c:811:16: error: call to undeclared function 'xsltGetProfileInformation'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
  811 |     c_retval = xsltGetProfileInformation(ctxt);
      |                ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg/libxslt-py.c:811:16: note: did you mean 'libxslt_xsltGetProfileInformation'?
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg/libxslt-py.c:801:1: note: 'libxslt_xsltGetProfileInformation' declared here
  801 | libxslt_xsltGetProfileInformation(PyObject *self ATTRIBUTE_UNUSED, PyObject *args) {
      | ^
  802 |     PyObject *py_retval;
  803 |     xmlDocPtr c_retval;
  804 |     xsltTransformContextPtr ctxt;
  805 |     PyObject *pyobj_ctxt;
  806 | 
  807 |     if (!PyArg_ParseTuple(args, (char *)"O:xsltGetProfileInformation", &pyobj_ctxt))
  808 |         return(NULL);
  809 |     ctxt = (xsltTransformContextPtr) PytransformCtxt_Get(pyobj_ctxt);
  810 | 
  811 |     c_retval = xsltGetProfileInformation(ctxt);
      |                ~~~~~~~~~~~~~~~~~~~~~~~~~
      |                libxslt_xsltGetProfileInformation
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg/libxslt-py.c:811:14: error: incompatible integer to pointer conversion assigning to 'xmlDocPtr' (aka 'struct _xmlDoc *') from 'int' [-Wint-conversion]
  811 |     c_retval = xsltGetProfileInformation(ctxt);
      |              ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg/libxslt-py.c:1456:5: error: call to undeclared function 'xsltSaveProfiling'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
 1456 |     xsltSaveProfiling(ctxt, output);
      |     ^
/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/libxslt/arm64-osx-dbg/libxslt-py.c:1606:16: error: call to undeclared function 'xsltTimestamp'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
 1606 |     c_retval = xsltTimestamp();
      |                ^
5 errors generated.
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "name": "joey",
  "version-string": "HEAD",
  "dependencies": [
    {
      "name": "libxslt",
      "default-features": false,
      "features": [
        "python"
      ]
    }
  ]
}

```
</details>

--- a/ports/libxslt/vcpkg.json
+++ b/ports/libxslt/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "libxslt",
   "version": "1.1.42",
+  "port-version": 1,
   "description": "Libxslt is a XSLT library implemented in C for XSLT 1.0 and most of EXSLT",
   "homepage": "https://github.com/GNOME/libxslt",
   "license": null,
@@ -39,7 +40,16 @@
     },
     "python": {
       "description": "Builds with python support",
-      "supports": "!windows"
+      "supports": "!windows",
+      "dependencies": [
+        {
+          "name": "libxslt",
+          "default-features": false,
+          "features": [
+            "profiler"
+          ]
+        }
+      ]
     },
     "thread": {
       "description": "Enable multi-threading support"

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -5474,7 +5474,7 @@
     },
     "libxslt": {
       "baseline": "1.1.42",
-      "port-version": 0
+      "port-version": 1
     },
     "libxt": {
       "baseline": "1.3.0",

--- a/versions/l-/libxslt.json
+++ b/versions/l-/libxslt.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "24f184fa7f0f8b396655fd08a4593c656811008a",
+      "version": "1.1.42",
+      "port-version": 1
+    },
+    {
       "git-tree": "8bd8a88f0bf38f7e3b7dda75aa505d413a496836",
       "version": "1.1.42",
       "port-version": 0

